### PR TITLE
Revise homepage logic to get first two articles from first issue

### PR DESCRIPTION
**Pull request by rlskoeser**
Revised homepage template to get the first issue, and then get the first two pages from that issue by order.

(We may have to tweak slightly when we add a second issue, since I don't know how the issues are sorted by default.)

### CODE REVIEW

**Comment by gwijthoff** [comment]
Hmm, when I serve this branch locally, I'm not getting the DBV card or the opening lines of the issue intro.

Screen Shot 2020-10-22 at 1 06 31 PM

File: themes/startwords/layouts/index.html [hunk 1]
```diff
@@ -14,19 +14,18 @@ <h1 class="sr-only">{{ .Site.Title }}</h1>
...
     <div class="issue-summary inverted grid">
         <div class="container">
-            {{ range first 1 (where .Site.AllPages "Type" "issue") }}
-            <span class="number">Issue {{ .Params.number }}</span>
-            <h2 class="theme">{{ .Params.theme }}</h2>
+            {{ $issue := index (where .Site.AllPages "Type" "issue") 0 }}
+            <span class="number">Issue {{ $issue.Params.number }}</span>
+            <h2 class="theme">{{ $issue.Params.theme }}</h2>
             <p class="summary">
                 {{ .Summary | plainify }}
```
Comment (thatbudakguy) [inline themes/startwords/layouts/index.html:21]: change this to `{{ $issue.Summary | plainify }}` to bring back the issue summary.

**Review by thatbudakguy — COMMENTED** [review]
I didn't see the issue @gwijthoff had with the DBV preview disappearing, but I did see the missing issue summary:

Screen Shot 2020-10-22 at 1 52 09 PM

Confirmed my suspicions that a simple template change did the trick to fix it; see my other comment.

**Review by gwijthoff — APPROVED** [review]
Okay @rlskoeser looks good! You've probably already thought of this, but this branch no longer includes the copyedits I've incorporated into the .md of a couple articles now so let's make sure not to override them.

**Comment by rlskoeser** [comment]
> Confirmed my suspicions that a simple template change did the trick to fix it; see my other comment.

@thatbudakguy I don't see your other comment — help ?

**Comment by thatbudakguy** [comment]
@rlskoeser it's the only review comment I made on this PR. grant already pushed a commit with the fix to this branch.

**Comment by rlskoeser** [comment]
🤦‍♀️  your comment was already resolved, wasn't obvious. I see it now. Thanks Grant for fixing it!

**Pull request merged**